packages/@lwc/errors/src/__tests__/errors.spec.ts

```diff
@@ -75,4 +77,22 @@ describe('error validation', () => {
 
         traverseErrorInfo(CompilerErrors, checkUniqueness, 'compiler');
     });
+
+    it('Next error code is updated in src/compiler/error-info/index.ts', () => {
```

Comment:
Nit: If someone updates the error code without seeing the comment to being with, they might be confused without a little more context.
```suggestion
    it('"Next error code: XXXX" comment is updated in src/compiler/error-info/index.ts', () => {
```